docs/source/re_manager_api.rst

```diff
@@ -0,0 +1,809 @@
+Parameters    **option**: *'immediate'* or *'deferred'*
+                  pause the plan immediately (roll back to the previous checkpoint) or continue
+                  to the next checkpoint.
+------------  -----------------------------------------------------------------------------------------
+Returns       **success**: *boolean*
+                  indicates if the request was processed successfully.
+
+              **msg**: *str*
+                  error message in case of failure, empty string ('') otherwise.
+------------  -----------------------------------------------------------------------------------------
+Execution     The request only initiates the operation of pausing the plan. Wait until the plan is
+              paused by polling 'manager_state' status field (expected value is 'paused').
+============  =========================================================================================
+
+
+.. _method_re_resume_stop_abort_halt:
+
+**'re_resume', 're_stop', 're_abort', 're_halt'**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+============  =========================================================================================
+Method        **'re_resume'**, **'re_stop'**, **'re_abort'**, **'re_halt'**
+------------  -----------------------------------------------------------------------------------------
+Description   Request Run Engine to resume, stop, abort or halt a paused plan. Fails if RE Worker
+              environment does not exist or 'manager_state' status field value is not 'paused'.
+------------  -----------------------------------------------------------------------------------------
+Parameters    ---
+------------  -----------------------------------------------------------------------------------------
+Returns       **success**: *boolean*
+                  indicates if the request was processed successfully.
+
+              **msg**: *str*
+                  error message in case of failure, empty string ('') otherwise.
+------------  -----------------------------------------------------------------------------------------
+Execution     The request only initiates the operation. Wait until the plan is paused by monitoring
+              'manager_state' status field (expected value is 'executing_queue' if execution is
+              resumed or 'idle' if execution was stopped).
+============  =========================================================================================
+
+
+.. _method_manager_stop:
+
+**'manager_stop'**
+^^^^^^^^^^^^^^^^^^
+
+
+============  =========================================================================================
+Method        **'manager_stop'**
+------------  -----------------------------------------------------------------------------------------
+Description   Exit RE Manager application. Clients will probably not need to initiate exit remotely,
+              but ability to do so it is extremely useful for automated testing.
```

Comment:
"... to do so it is extremely ..." --> "... to do so is extremely ..."

Reply:
Changes implemented in PR https://github.com/bluesky/bluesky-queueserver/pull/118